service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/gceinstance/CreateGceInstanceStep.java

```diff
@@ -193,7 +194,15 @@ static Instance setFields(
                 .setInitializeParams(
                     new AttachedDiskInitializeParams()
                         .setSourceImage(creationParameters.getVmImage())
-                        .setDiskSizeGb(Long.valueOf(100)))));
+                        .setDiskSizeGb(Long.valueOf(100))),
+            new AttachedDisk()
+                .setBoot(false)
+                .setDeviceName(DATA_DISK_NAME)
+                .setAutoDelete(true)
+                .setInitializeParams(
+                    new AttachedDiskInitializeParams()
+                        .setDiskType(creationParameters.getDataDiskType())
```

Comment:
should this be conditional? 

only add an AttachedDisk if the creationParameters contains datadisktype and datadisksizegb

Reply:
My impression from John's doc was that all instances would have an extra PD... @jmczerk confirm?

Reply:
or a default type and size?

Reply:
Hm, but GCP will default their values if none are provided.